test/server/server_fuzz_test.cc

```diff
@@ -16,6 +16,39 @@
 namespace Envoy {
 namespace Server {
 
+void makePortHermetic(envoy::api::v2::core::Address& address) {
+  if (address.has_socket_address()) {
+    address.mutable_socket_address()->set_port_value(0);
+  }
+}
+
+envoy::config::bootstrap::v2::Bootstrap
+makeHermeticPathsAndPorts(Fuzz::PerTestEnvironment& test_env,
+                          const envoy::config::bootstrap::v2::Bootstrap& input) {
+  envoy::config::bootstrap::v2::Bootstrap output;
+  output.MergeFrom(input);
```

Comment:
Why not just `envoy::config::bootstrap::v2::Bootstrap output(input)` to get the initial state? Isn't this equivalent?